ReminderApp-OBJC/ReminderApp-OBJC/Classes/ReminderCell/CDEditPriorityCell.h

```diff
@@ -0,0 +1,25 @@
+//
+//  CDEditPriorityCell.h
+//  ReminderApp-OBJC
+//
+//  Created by Catalin David on 19/07/16.
+//  Copyright © 2016 Catalin David. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "CDReminder.h"
+
+typedef NS_ENUM(NSInteger, CDPriority){
```

Comment:
This should go in the Reminder class, it is related to the model. If you need this enum in some other class you will have to import CDEditPriorityCell.h which is not ok.